dev/com.ibm.ws.session.cache/resources/com/ibm/ws/session/store/cache/resources/WASSessionCache.nlsprops

```diff
@@ -19,4 +19,44 @@
+
+ERROR_REMOVING_SESSION=SESN0301E: An exception occurred when removing a session from the cache. The exception is: {0}.
+ERROR_REMOVING_SESSION.explanation=An operation against the cache failed due to the specified exception.
+ERROR_REMOVING_SESSION.useraction=See the exception for more details. The JCache provider's configuration might need to be updated.
+
+PROP_HIT_ERROR=SESN0302E: An exception occurred when storing application data changes to the cache. The exception is: {0}.
+PROP_HIT_ERROR.explanation=An operation against the cache failed due to the specified exception.
+PROP_HIT_ERROR.useraction=See the exception for more details. The JCache provider's configuration might need to be updated.
+
+STORE_SESS_ERROR=SESN0303E: An exception occurred when storing a session in the cache. The exception is: {0}.
+STORE_SESS_ERROR.explanation=An operation against the cache failed due to the specified exception.
+STORE_SESS_ERROR.useraction=See the exception for more details. The JCache provider's configuration might need to be updated.
+
+LOAD_VALUE_ERROR=SESN0304E: An exception occurred when reading in an object of the application data for a session from the cache. The exception is: {0}.
+LOAD_VALUE_ERROR.explanation=An operation against the cache failed due to the specified exception.
+LOAD_VALUE_ERROR.useraction=See the exception for more details. The JCache provider's configuration might need to be updated.
+
+ERROR_CACHE_ACCESS=SESN0305E: An exception occurred while attempting to access the cache. The exception is: {0}.
+ERROR_CACHE_ACCESS.explanation=An operation against the cache failed due to the specified exception.
+ERROR_CACHE_ACCESS.useraction=See the exception for more details. The JCache provider's configuration might need to be updated.
+
+ERROR_SESSION_INVAL=SESN0306E: An exception occurred when invalidating a session in the cache. The exception is: {0}.
+ERROR_SESSION_INVAL.explanation=An operation against the cache failed due to the specified exception.
+ERROR_SESSION_INVAL.useraction=See the exception for more details. The JCache provider's configuration might need to be updated.
+
+ERROR_SESSION_INIT=SESN0307E: An exception occurred when initializing the cache. The exception is: {0}.
+ERROR_SESSION_INIT.explanation=An operation against the cache failed due to the specified exception.
+ERROR_SESSION_INIT.useraction=See the exception for more details. The JCache provider's configuration might need to be updated.
+
+WARN_CONFIG_INVALID_HTTPSESSIONCACHE=SESN0308W: Invalid or missing httpSessionCache configuration detected. {0}
+WARN_CONFIG_INVALID_HTTPSESSIONCACHE.explanation=The sessionCache feature was unable to fully initialize due to a missing or invalid configuration.
+WARN_CONFIG_INVALID_HTTPSESSIONCACHE.useraction=Verify the server configuration defines a httpSessionCache that includes a libraryRef attribute.
+
+ERROR_CONFIG_EMPTY_LIBRARY=SESN0309E: The session cache library {0} is empty. {1}
```

Comment:
I am not 100% sure what {0} is, but if it is like the name of the session cache library or something, it should be before the session cache library: "The {0} session cache library..."